markdown/pxf/ReadWritePXF.html.md.erb

```diff
@@ -131,6 +149,8 @@ Note: The <code class="ph codeph">DELIMITER</code> parameter is mandatory.
 </tbody>
 </table>
 
+**Notes**: Metadata identifies the Java class that provides field definitions in the relation. OutputFormat identifies the file format for which a specific profile is optimized. While the built-in `Hive*` profiles provide Metadata and OutputFormat classes, most profiles will have no need to implement or specify these classes.
```

Comment:
We can probably mention that PXF service can produce data in different formats - TEXT, GPDBWritable, and outputFormat property means given profile optimized for particular output format(TEXT/GPDBWritable).